### ISSUE TITLE
Community Handbook issues

### ISSUE THREAD

**Issue by da5nsy** [alan-turing-institute/the-turing-way#2674]
### Description

Subset of #2611 relating to the community handbook.

There seems to be 2 issues:
1. Referencing in the toc
```
←[91mC:\Users\cege-user\Documents\the-turing-way\book\website\community-handbook\community-handbook.md:: WARNING: toctree contains reference to nonexisting document 'community-handbook/contributing' [etoc.ref]←[39;49;00m
←[91mC:\Users\cege-user\Documents\the-turing-way\book\website\community-handbook\contributing\contributing-templates.md: WARNING: document isn't included in any toctree←[39;49;00m
←[91mC:\Users\cege-user\Documents\the-turing-way\book\website\community-handbook\contributing\contributing-workflow.md: WARNING: document isn't included in any toctree←[39;49;00m
```

2. undefined label: contributors-record-contributors
```
←[91mC:\Users\cege-user\Documents\the-turing-way\book\website\community-handbook\acknowledgement.md:19: WARNING: undefined label: contributors-record-contributors←[39;49;00m
←[91mC:\Users\cege-user\Documents\the-turing-way\book\website\community-handbook\acknowledgement\acknowledgement-members.md:12: WARNING: undefined label: contributors-record-contributors←[39;49;00m
←[91mC:\Users\cege-user\Documents\the-turing-way\book\website\community-handbook\acknowledgement\acknowledgement-record.md:4: WARNING: undefined label: contributors-record-contributors←[39;49;00m
```

**Comment by da5nsy** [comment]
Looks like the undefined label is due to some misformatting introduced in https://github.com/alan-turing-institute/the-turing-way/commit/f0d4c7b423dceb9f55c73627dc07e9d40e424963 (lines 1159 - 1162, and separately line 1624).

@malvikasharan - Could you have a look at this?

**Comment by da5nsy** [comment]
Fixed via #2689 and #2724
